addons/voice/org.openhab.voice.gtts/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,38 @@
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ManifestVersion: 2
+Bundle-Name: Google Cloud TTS Service
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.voice.gtts;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Bundle-ClassPath: 
+ .,
+ lib/grpc-core-1.12.0.jar,
+ lib/grpc-netty-shaded-1.12.0.jar,
+ lib/grpc-auth-1.12.0.jar,
+ lib/proto-google-cloud-texttospeech-v1beta1-0.18.0.jar,
+ lib/grpc-stub-1.12.0.jar,
+ lib/grpc-context-1.12.0.jar,
+ lib/api-common-1.6.0.jar,
+ lib/gax-1.28.0.jar,
+ lib/gax-grpc-1.28.0.jar,
+ lib/grpc-protobuf-lite-1.12.0.jar,
+ lib/grpc-protobuf-1.12.0.jar,
+ lib/opencensus-api-0.11.0.jar,
+ lib/opencensus-contrib-grpc-metrics-0.11.0.jar,
+ lib/google-auth-library-oauth2-http-0.9.1.jar,
+ lib/google-auth-library-credentials-0.9.1.jar,
+ lib/protobuf-java-3.5.1.jar,
+ lib/google-http-client-jackson2-1.19.0.jar,
+ lib/guava-24.1-jre.jar,
+ lib/threetenbp-1.3.6.jar,
+ lib/google-http-client-1.23.0.jar,
+ lib/google-cloud-texttospeech-0.53.0-beta.jar,
+ lib/jackson-core-2.8.8.jar
+Import-Package: 
+ org.eclipse.smarthome.config.core,
```

Comment:
The following import is missing here according to SAT:

`org.eclipse.jdt.annotation;resolution:=optional,`

Reply:
Done